### ISSUE TITLE
weighting documentation does not list maintainer (Trac #1236)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1230]
Migrated from https://code.icecube.wisc.edu/ticket/1236
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo core", 
    "summary": "weighting documentation does not list maintainer", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-20T08:27:00", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150921T12:03:33: 
* jvansanten commented:

> In changeset 137790:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137790"
> This project is mine, for better or worse. Fixes #1230.
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
